azure-pipelines.yml

```diff
@@ -161,7 +161,7 @@ stages:
             python.installer.name: 'macos11.pkg'
           Python311:
             python.version: '3.11'
-            python.org.version: '3.11.0'
+            python.org.version: '3.11.1'
```

Comment:
sometime 3.11.0 and sometime 3.11.1?

Reply:
This is installed directly from ftp.python.org (official Python) and is for Azure.  `3.11.0` you are referring to is for CircleCI and is installed via pyenv